src/content/docs/apm/new-relic-apm/getting-started/introduction-apm.mdx

```diff
@@ -16,150 +16,172 @@ redirects:
   - /docs/apm
   - /docs/apm/new-relic-apm
   - /docs/apm 
+  - /docs/apm/new-relic-apm/getting-started/introduction-apm
 ---
 
-import apmScreenshot from 'images/apm-screenshot.png'
+import apmOverview from 'images/apm-intro-overview.png'
```

Comment:
You had mentioned in a PR I committed today that's it's best to use the same filename when we replace a screenshot.